projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -79,9 +79,17 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     protected readonly renderingStrategy: RenderingStrategy,
     protected readonly svgUtilService: SvgUtilService,
     protected readonly d3Utils: D3UtilService,
-    protected readonly domRenderer: Renderer2
+    protected readonly domRenderer: Renderer2,
+    protected readonly groupId?: string
   ) {}
 
+  // tslint:disable-next-line
```

Comment:
Do not disable lint rules out of convenience - if it needs to be disabled, there needs to be a reason - and no-any never has a good reason.